tests/dump-cert-chain-to-text.t

```diff
@@ -200,6 +200,8 @@ Dump a live cert chain (squareup-chain.crt)
   Extended Key Usage:
   \tClient Auth (esc)
   \tServer Auth (esc)
+  Warnings:
+  \t[w_sub_ca_aia_does_not_contain_issuing_ca_url]  (esc)
```

Comment:
We shouldn't expose these internal warning types, probably. Ideally, it would be something like `[RFC5280] Nice description`.